src/lineup/ARankingView.ts

```diff
@@ -287,8 +287,21 @@ export abstract class ARankingView extends AView {
     colDesc.color = this.colors.getColumnColor(id);
     const accessor = createAccessor(colDesc);
 
+    // find last index of the current selected ID
```

Comment:
have you considered adding some logic to the (multi) selection adapter itself? in the end this is the one that cares about the order

Reply:
yes, I've considered that, but I didn't see a better place, since I need to find the previous column after which I want to insert my current column. As far as I have seen it I only have column descriptions before
Can you point me to a place which would be better to sort the columns beforehand?